### PR TITLE
Python rewrite Work In Progress

### PR DESCRIPTION
**Pull request by stevengborrego**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/venv/
```
Comment (ryanpdx) [inline .gitignore:1]: Replace with the default python gitignore: https://github.com/github/gitignore/blob/main/Python.gitignore

File: stationd.py [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+
+
+class Invalid_Command(Exception):
+    def __init__(self, command_obj):
+        self.command_obj = command_obj
+
+    def send_response(self):
+        message = f'FAIL: Invalid Command\n'
+        self.command_obj.sock.sendto(message.encode('utf-8'), self.command_obj.addr)
+        log(self.command_obj, message)
```
Comment (ryanpdx) [inline stationd.py:545]: Make a base exception for this project (something like `StationdError`) and derived all other custom exception from that one.

File: stationd.py [hunk 3]
```diff
@@ -0,0 +1,554 @@
+"""
```
Comment (ryanpdx) [inline stationd.py:1]: Convert this into a `uniclogs_stationd` python module with multiple files.

**Review by ryanpdx — COMMENTED** [review]
This is some general python project feedback. Most of this is to help with maintainability and readability.

Other things to do:
- Add a `requirements.txt` and all the project python dependencies to it. 
- Copy the `.flake8` config file from `oresat-olaf` repo.
- Add sphinx docstrings to everything.
- Adding type annotations to function definitions are really helpful

**Comment by aSmig** [comment]
There are still some good notes in this PR for things that can be improved. Please add issues for anything that appears to be outstanding.

**Pull request merged**